src/main/java/SortArray.java

```diff
@@ -0,0 +1,53 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.Random;
+
+public class SortArray {
+    public static void main(String[] args) {
+        Scanner s1 = new Scanner(System.in);
+        System.out.print("Input array size: ");
+        int arraySize = s1.nextInt();
+
+        //Generate a random array of integers of provided size and print it
+        int arr[] = new int[arraySize];
+        Random randomNumber = new Random();
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = randomNumber.nextInt(1000);
+            System.out.println(arr[i]);
+        }
+
+        System.out.println(Arrays.toString(bubble(arr)));
+        System.out.println(Arrays.toString(selection(arr)));
+    }
+
+    public static int[] bubble(int[] arrToSort) {
+        int count = arrToSort.length * arrToSort.length;
+        do {
```

Comment:
I'm not sure if the do .. while loop fits here well, since it's commonly used when you have to do at least one iteration and then decide if it's required to proceed. `for` loop is better here since it became more readable.